code/modules/mob/living/basic/space_fauna/bear/bear.dm

```diff
@@ -36,26 +31,23 @@
 	friendly_verb_continuous = "bear hugs"
 	friendly_verb_simple = "bear hug"
 
-	//Space bears aren't affected by cold.
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 0
-	maxbodytemp = 1500
-
 	faction = list(FACTION_RUSSIAN)
 
-	footstep_type = FOOTSTEP_MOB_CLAW
-
+	habitable_atmos = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minimum_survivable_temperature = 0
```

Comment:
i dont understood.  there an example i can learend from?

Reply:
![image](https://github.com/tgstation/tgstation/assets/34697715/a7638b6f-817a-4176-8800-e221d1b9c69c)

This is what we do for space lizards, instead of having objective numbers, we relate the numbers to the defines we have for `TCMB` and `T0C` (which are used heavily in atmospherics code and that correlation to how mobs deal with atmospherics) in case we need to alter any of those defines in the future (extremely unlikely, but will be quite helpful should it ever arise)

Reply:
thanks